crates/rpc/rpc/src/eth/api/pending_block.rs

```diff
@@ -112,6 +171,19 @@ impl PendingBlockEnv {
             // append transaction to the list of executed transactions
             executed_txs.push(tx.into_signed());
         }
+        // executes the withdrawals and commits them to the Database and BundleState.
+        let balance_increments = post_block_withdrawals_balance_increments(
+            &chain_spec,
+            block_env
+                .timestamp
+                .to_string()
+                .parse::<u64>()
+                .expect("timestamp should be converted into u64"),
```

Comment:
hmm, I don't actually know why timestamp is a u256,

but we can use `.try_into().unwrap_or(u64::Max)` this will always succeed because timestamp is a u64